src/index.js

```diff
@@ -161,10 +161,6 @@ checkboxes.forEach((chbox) => {
   });
 });
 
-// function editTask() { // this is function for edit task
-//   document.getElementsByClassName('description').contentEditable = true;
-// }
-
 const editTask = index => {
   todoItemElems[index].contentEditable = true;
 };
```

Comment:
- [ ] As you can see here this `editTask` method is not associated with an event listener nor has it a callback in any event listener function. You can notice your `addTask` method is working properly but `editTask`, `deleteTask` and `completeTask` functions are not working. You can see the attached screenshot for the returned error.
 
![Screenshot from 2021-08-01 05-42-05](https://user-images.githubusercontent.com/64894927/127754929-c2d62609-3d00-41ff-8c0e-2f23cdf72a00.png)

Reply:
Dear @henatan99,

Are you check8bg the live demo https://elmar8287.github.io/todo-list/  all the mentioned functions are working.

Reply:
I am checking it. Just give me a minute.

Reply:
Dear @elmar8287, I have checked the live demo and the `deleteTask` and `completeTask` are working properly. But still, the `editTask` method does not update the local storage as it returns to its initial state when the page is refreshed. In addition, the `Clear all completed` button is not working at all and I don't think you have implemented the method. 
Honestly speaking I don't know why it is not working in the localhost and it is working on github.io but I also noticed those methods ( `editTasks, ...) are just pure functions and are not associated with an event. You can correct me if I missed something. 
My suggestion, for now, is just to work on the issues which are failing on the live demo also (`editTask` and `clearAllCompleted` methods). After making these changes, kindly maks a request for another review. I wish you good luck! :pray:

Reply:
Dear @henatan99,

This is my last review ask attempt. Also this is my week repeating.

Reply:
Dear @henatan99! 🥇 

Many thanks for the review. I appreciate your comments 👍 

Many thanks and regards,

Elmar